pkg/drivers/kic/kic.go

```diff
@@ -347,13 +347,20 @@ func (d *Driver) Stop() error {
 		}
 		if len(containers) > 0 {
 			if err := runtime.StopContainers(containers); err != nil {
-				glog.Errorf("unable to stop containers : %v", err)
+				glog.Infof("unable to stop containers : %v", err)
```

Comment:
Maybe a Warningf?

Reply:
I leave the warrning for the second one (KillContainers) so we dont scare the user too early.
if we can not kill the containers then will warn them.